packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -125,6 +125,7 @@ couple of the tasks available are:
 * `tasks.InvokeActivity` -- start an Activity (Activities represent a work
   queue that you poll on a compute fleet you manage yourself)
 * `tasks.InvokeFunction` -- invoke a Lambda function with function ARN
+* `tasks.InvokeBatchJob` -- invoke a Batch job
```

Comment:
Further to this, we need a section with a code snippet on how to invoke the batch job. See below the sections for the other types.

Reply:
Will add a code snippet based on the final revision that we have here.

Reply:
Also, just noticed batch has a much better API support in v1.26.0. Will make the use of it right away.